distribution/docs/src/main/resources/_contents/_endpoints/application-endpoint-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+==== Application Upload Endpoint
+
+The Application Upload Endpoint enables uploading new and upgraded applications to the system.
+
+===== Installing Application Upload Endpoint
+
+The Application Upload Endpoint is installed by default with a standard installation as part of the ${ddf-admin} application.
+
+===== Application Upload Endpoint
+
+The Application endpoint is available at <${secure_url}/services/application.
+
+===== Configuring Application Upload Endpoint
+
+No configuration necessary.
+
+===== Known Issues with Application Endpoint
```

Comment:
Instead of having individual _Known Issues_ sections, should those be all be reported as bugs in the release notes if they actually are problems and rename this section _Usage Limitations_ or something similar?